src/views/layout.njk

```diff
@@ -23,9 +23,9 @@
 {% block header %}
   {{
     govukHeader({
-      homepageUrl: paths.ROOT_URI,
+      homepageUrl: Paths.ROOT_URI + '/',
```

Comment:
The story called for the header - does the story also include the crown icon? Should it not redirect to more 'globa' ch gov uk service? @peadarkelly

Reply:
Thought this too - checked Appeals in Jayti's absence and it goes to the Appeals Landing Page. Minimal fix if Jayti wishes for it to go to the CH homepage instead.

Reply:
Ok, green light from me then.